### ISSUE TITLE
Accordion overview

### ISSUE THREAD

**Issue by maxokorokov** [ng-bootstrap/ng-bootstrap#3104]
Add the overview page for accordion.

Ex. Header customization is often a problem:

Full clickable header in order of preference (stacklBlitz → https://stackblitz.com/edit/angular-xcm5fq?file=app/accordion-header.html
):
1. Since bootstrap `4.3`, use `.stretched-link` (https://getbootstrap.com/docs/4.3/utilities/stretched-link/). The only caveat is that card header should be `position: relative`

```html
<ngb-accordion>
  <ngb-panel>
    <ng-template ngbPanelHeader>
        <button ngbPanelToggle class="btn btn-link stretched-link">Toggle</button>
    </ng-template>
    <ng-template ngbPanelContent>...</ng-template>
  </ngb-panel>
</ngb-accordion>
```

2. Use the standard markup, but customize the button, simple example:
```html
<ngb-accordion>
  <ngb-panel>
    <ng-template ngbPanelHeader>
        <button ngbPanelToggle class="btn btn-link w-100 text-left">Toggle</button>
    </ng-template>
    <ng-template ngbPanelContent>...</ng-template>
  </ngb-panel>
</ngb-accordion>
```

2. Don't use the button, put whatever (loosing a11y, disabling in this case), style however you want:
```html
<ngb-accordion #acc="ngbAccordion">
  <ngb-panel id="myPanel">
    <ng-template ngbPanelHeader>
        <div (click)="acc.toggle('myPanel')">Toggle</div>
    </ng-template>
    <ng-template ngbPanelContent>...</ng-template>
  </ngb-panel>
</ngb-accordion>
```

**Comment by Devqon** [comment]
I have used the last option in my project, because I have a custom button in the header, which should do other actions.

**Comment by stguitar** [comment]
@maxokorokov Are you opening this to track and suggest a change to docs to show how these 3 options could be applied?  If that is the case, I could probably take a crack at it.

**Comment by maxokorokov** [comment]
@stguitar yeah, just to track / not forget about it. 

If you're up to it, please do, you can inspire by pagination:
- https://ng-bootstrap.github.io/#/components/pagination/overview
- https://github.com/ng-bootstrap/ng-bootstrap/tree/master/demo/src/app/components/pagination/overview
- `yarn demo` is to deploy demo locally on `localhost:4200`

There is no particular list of topics, just the way to customize header for now, so I guess:
- basic usage
- header customization
are the two topics we could have to start

**Comment by stguitar** [comment]
Cool! Yea I’m up for helping.  I’ll let you know how it goes.

**Comment by mtpultz** [comment]
Using option 2 worked well for my use case since Bootstrap 4.3 wasn't an option, but couldn't rely only on utility classes from Bootstrap since I needed to adjust the padding on the `.card-header` to have the panel header be completely clickable.

```
ngb-accordion .card-header {
  // Make the entire accordion header clickable by
  // 1) Removing padding from header
  padding: 0;

  & > button {
    // 2) And adding more padding to the button until satisfied
    padding: 1rem 1.5rem;

    &:focus,
    &:hover {
      text-decoration: none;
    }
  }
}
```

**Comment by maxokorokov** [comment]
Fixed by #4481
